website/contents/pages.js

```diff
@@ -407,6 +407,14 @@ export const docPages = generatePath([
             name: 'GridCellLayer',
             content: getDocUrl('layers/grid-cell-layer.md')
           },
+          {
+            name: 'H3HexagonLayer',
+            content: getDocUrl('layers/h3-hexagon-layer.md')
```

Comment:
Nit: I think we should have a separate index rather than duplicate layers in two places just to get an alphabetic list. An index could be easily auto-generated by ocular once we port the deck website after v7 release. The docs should just contain folders corresponding to the modules. If keeping this.